OpenSim/Examples/ExampleHopperDevice/exampleHopperDevice.cpp

```diff
@@ -212,14 +212,15 @@ void run(bool showVisualizer, double finalTime)
     {
         // Build the hopper.
         auto hopper = buildHopper(showVisualizer);
+        hopper.finalizeFromProperties();
```

Comment:
Why was this added? It would be great if the internal steps of the construction/initializaton are not exposed and users only need to know initSystem.

Reply:
Looks like it was added while I was debugging. (See `exampleHopperDevice_answers.cpp` below was not modified). I will remove.